models/experimental/functional_squeezebert/tests/test_performance.py

```diff
@@ -0,0 +1,155 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import time
+
+import pytest
+
+from loguru import logger
+import torch
+import transformers
+
+
+import ttnn
+from ttnn.model_preprocessing import preprocess_model_parameters
+
+from models.experimental.functional_squeezebert.tt import ttnn_functional_squeezebert
+from models.experimental.functional_common.attention_mask_functions import get_extended_attention_mask
+from transformers import SqueezeBertForQuestionAnswering
+
+
+from models.utility_functions import (
+    skip_for_wormhole_b0,
+    enable_persistent_kernel_cache,
+    disable_persistent_kernel_cache,
+)
+from models.perf.perf_utils import prep_perf_report
+
+
+def preprocess_inputs(
+    input_ids,
+    token_type_ids,
+    position_ids,
+    attention_mask,
+):
+    batch_size, *_ = input_ids.shape
+
+    input_ids = ttnn.from_torch(input_ids, dtype=ttnn.uint32)
+    token_type_ids = ttnn.from_torch(token_type_ids, dtype=ttnn.uint32)
+    position_ids = ttnn.from_torch(position_ids, dtype=ttnn.uint32)
+
+    if attention_mask is not None:
+        attention_mask = get_extended_attention_mask(attention_mask, input_ids.shape)
+        attention_mask = attention_mask.expand((batch_size, -1, -1, -1))
+        attention_mask = torch.clamp(attention_mask, min=-100000)
+        attention_mask = ttnn.from_torch(
+            attention_mask,
+            dtype=ttnn.bfloat16,
+            layout=ttnn.TILE_LAYOUT,
+        )
+    return input_ids, token_type_ids, position_ids, attention_mask
+
+
+def get_expected_times(bert):
+    return {
+        ttnn_functional_squeezebert: (7.97, 8.32),
+    }[bert]
+
+
+@skip_for_wormhole_b0()
+@pytest.mark.models_performance_bare_metal
```

Comment:
Are you trying to run e2e performance test? It's not specified in any scripts

Reply:
Updated `run_performance.sh` with e2e performance test along with device perf test.